config/vufind/Demo.ini

```diff
@@ -46,7 +46,7 @@ services[] = 'custom'
 ; This setting can be used to create fake checked out items for specific records.
 ; The value is a JSON document representing the status information returned by the
 ; driver.
-;transactions = '[{"id":"1234", ... "renewable": true}]';
+transactions = '[{"id":"1234", ... "renewable": true}]';
```

Comment:
This looks like an unintentional commit.

Reply:
Sorry, I should have seen this myself. Reverted it.